### DISCUSSION TITLE
[Feature] Option to backup all folders on mobile app. So new albums/folders are automatically included.

### DISCUSSION THREAD

**Discussion by Cookie-Monster-Coder** [immich-app/immich#9406] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Synology photos has an option in the backup settings to backup all folders. When this is selected, any new albums/folders that are created are automatically selected and backed up without needing to manually go back into the settings and select them.

![Screenshot_20240512_090323_Photos](https://github.com/immich-app/immich/assets/53286741/1b385e2b-cb34-492b-bcf4-985d7d8e4b33)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by mmomjian** [comment]
This is already supported. On both Android and iOS, choosing to backup "Recent" will backup all images that are added after that time, even if added to an album.

**Reply by Cookie-Monster-Coder** [reply]
I must have missed something when testing this the first time, seems to be working as intended when testing again. Thank you!
